node/src/actors/chain_manager/mining.rs

```diff
@@ -898,11 +899,11 @@ fn build_block(
                 vt_tx.hash(),
             );
             value_transfer_txns.push(vt_tx.clone());
-            transaction_fees += transaction_fee;
-            vt_weight += transaction_weight;
+            transaction_fees = transaction_fees.saturating_add(transaction_fee);
+            vt_weight = new_vt_weight;
         }
 
-        if new_vt_weight > max_vt_weight - min_vt_weight {
+        if vt_weight > max_vt_weight.saturating_sub(min_vt_weight) {
```

Comment:
Sorry, what is this line for? I don't understand

Reply:
If the free space in the block for VTTransactions is less than the minimum weight for a VTTransactions, it not make sense to continue checking the rest of pending transactions